CI fails in wheel workflow

<!-- ⚠️ If you do not respect this template, your issue will be closed -->
<!-- ⚠️ Make sure to browse the opened and closed issues -->

### Informations

- **Qiskit Aer version**:
- **Python version**:
- **Operating system**:

### What is the current behavior?

CI fails to build spdlog.

```
2022-07-22T08:28:53.3639273Z   spdlog/1.5.0: Configuring sources in C:\Users\runneradmin\.conan\data\spdlog\1.5.0\_\_\source
2022-07-22T08:28:53.3639588Z   
2022-07-22T08:28:53.3639841Z   spdlog/1.5.0: Copying sources to build folder
2022-07-22T08:28:53.3640275Z   spdlog/1.5.0: Building your package in C:\Users\runneradmin\.conan\data\spdlog\1.5.0\_\_\build\10066f31da826d912bc30f8d5a09a091b81dd02b
2022-07-22T08:28:53.3640683Z   spdlog/1.5.0: Generator cmake created conanbuildinfo.cmake
2022-07-22T08:28:53.3641032Z   spdlog/1.5.0: Generator cmake_find_package created Findfmt.cmake
2022-07-22T08:28:53.3641424Z   spdlog/1.5.0: Generator cmake_find_package_multi created fmt-config-version.cmake
2022-07-22T08:28:53.3642083Z   spdlog/1.5.0: Generator cmake_find_package_multi created fmtTarget-release.cmake
2022-07-22T08:28:53.3642549Z   spdlog/1.5.0: Generator cmake_find_package_multi created fmtTargets.cmake
2022-07-22T08:28:53.3642956Z   spdlog/1.5.0: Generator cmake_find_package_multi created fmt-config.cmake
2022-07-22T08:28:53.3643289Z   spdlog/1.5.0: Aggregating env generators
2022-07-22T08:28:53.3643530Z   spdlog/1.5.0: Calling build()
2022-07-22T08:28:53.3643836Z   spdlog/1.5.0:
2022-07-22T08:28:53.3644213Z   WARN: Remotes registry file missing, creating default one in C:\Users\runneradmin\.conan\remotes.json
2022-07-22T08:28:53.3644721Z   fmt/6.2.1: WARN: Using the new toolchains and generators without specifying a build profile (e.g: -pr:b=default) is discouraged and might cause failures and unexpected behavior
2022-07-22T08:28:53.3645219Z   spdlog/1.5.0: ERROR: Package '10066f31da826d912bc30f8d5a09a091b81dd02b' build failed
2022-07-22T08:28:53.3645685Z   spdlog/1.5.0: WARN: Build folder C:\Users\runneradmin\.conan\data\spdlog\1.5.0\_\_\build\10066f31da826d912bc30f8d5a09a091b81dd02b
2022-07-22T08:28:53.3646104Z   ERROR: spdlog/1.5.0: Error in build() method, line 112
2022-07-22T08:28:53.3646915Z   	if tools.Version(self.version) < "1.7" and tools.Version(self.deps_cpp_info["fmt"].version) >= "7":
2022-07-22T08:28:53.3647288Z   	ConanException: Invalid version 'None'
2022-07-22T08:28:53.3647613Z   CMake Error at cmake/conan.cmake:402 (message):
2022-07-22T08:28:53.3647858Z     Conan install failed='1'
2022-07-22T08:28:53.3648129Z   Call Stack (most recent call first):
2022-07-22T08:28:53.3648436Z     cmake/conan.cmake:497 (conan_cmake_install)
2022-07-22T08:28:53.3648698Z     cmake/conan_utils.cmake:71 (conan_cmake_run)
2022-07-22T08:28:53.3649030Z     cmake/dependency_utils.cmake:20 (setup_conan)
2022-07-22T08:28:53.3649337Z     CMakeLists.txt:145 (setup_dependencies)
2022-07-22T08:28:53.3649599Z   
2022-07-22T08:28:53.3649911Z   
2022-07-22T08:28:53.3650184Z   -- Configuring incomplete, errors occurred!
```

### Steps to reproduce the problem

Errors in wheel workflow.

### What is the expected behavior?

Pass CI.

### Suggested solutions

--- a/cmake/conan_utils.cmake
+++ b/cmake/conan_utils.cmake
@@ -10,7 +10,7 @@ macro(setup_conan)
     # Right now every dependency shall be static
     set(CONAN_OPTIONS ${CONAN_OPTIONS} "*:shared=False")
 
-    set(REQUIREMENTS nlohmann_json/3.1.1 spdlog/1.5.0)
+    set(REQUIREMENTS nlohmann_json/3.1.1 spdlog/1.9.2)
     list(APPEND AER_CONAN_LIBS nlohmann_json spdlog)
     if(APPLE AND CMAKE_CXX_COMPILER_ID MATCHES "Clang")
         list(APPEND AER_CONAN_LIBS llvm-openmp)

